ports/freealut/vcpkg.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "freealut",
+  "version-string": "1.1.0",
+  "description": [
+    "FreeALUT is a free implementation of OpenAL's ALUT standard.",
+    "ALUT is a set of portable functions which remove the annoying details of getting an audio application started. It is the OpenAL counterpart of what GLUT is for OpenGL. "
+  ],
+  "homepage": "https://github.com/vancegroup/freealut",
+  "supports": "(x86 | x64) & (linux | osx | windows)",
```

```suggestion
  "supports": "!uwp & !arm & !arm64",
```
We prefer disables over enables, leaving the door open for new platforms